src/backend/distributed/master/master_stage_protocol.c

```diff
@@ -191,10 +191,14 @@ master_append_table_to_shard(PG_FUNCTION_ARGS)
 	text *sourceTableNameText = PG_GETARG_TEXT_P(1);
 	text *sourceNodeNameText = PG_GETARG_TEXT_P(2);
 	uint32 sourceNodePort = PG_GETARG_UINT32(3);
+
 	char *sourceTableName = text_to_cstring(sourceTableNameText);
 	char *sourceNodeName = text_to_cstring(sourceNodeNameText);
 
-	char *shardName = NULL;
+	Oid shardSchemaOid = 0;
```

Comment:
> There are 2 different schemas, table names etc. in the context of this function. One is the source table we read the data from and the other one is the target we will write data into. Therefore I prefixed them with; remote(if it is the source we read from) and shard(if it is the target we will write into)

Then, can we use source prefix for source instead of remote?